aep/0193.yaml

```diff
@@ -0,0 +1,44 @@
+rules:
+  aep-193-error-response-schema:
+    description: Error response body should contain all required fields according to AEP-193.
+    message: '{{error}}'
+    severity: warn
+    formats: ['oas2', 'oas3']
+    given: $.paths[*][*].responses[?(@property >= 400)].content[*].schema.properties
+    then:
+      function: schema
+      functionOptions:
+        schema:
+          type: object
+          required: ['type']
+          properties:
+            type:
+              type: object
+              required: ['type']
+              properties:
+                type:
+                  enum: ['string']
+            title:
+              type: object
+              required: ['type']
+              properties:
+                type:
+                  enum: ['string']
+            status:
```

Comment:
include minimum / maximum? https://aep.dev/json-schema/type/problems.json https://datatracker.ietf.org/doc/html/rfc9457#name-json-schema-for-http-proble